osquery/tables/applications/browser_chrome.cpp

```diff
@@ -54,6 +54,16 @@ static std::vector<fs::path> getChromePaths() {
     chromePaths.push_back("/.config/chromium/%/Extensions/");
   }
 
+  if (isPlatform(PlatformType::TYPE_WINDOWS)) {
+    chromePaths.push_back(
+        "\\AppData\\Local\\Yandex\\YandexBrowser\\User Data\\%\\Extensions\\");
+  } else if (isPlatform(PlatformType::TYPE_OSX)) {
+    chromePaths.push_back(
+        "/Library/Application Support/Yandex/YandexBrowser/%/Extensions/");
+  } else {
+    chromePaths.push_back("/.config/yandex-browser%/%/Extensions/");
```

Comment:
Is the `yandex-browser%` wildcard intentional here?

Reply:
From the PR:
> For Linux systems the default path /.config/yandex-browser%/%/Extensions/ has the first %-sign because currently it's yandex-browser-beta. Thus, in case of future changes we don't need to fix the code one more time.

Reply:
Yeah, exactly. This is intentional.

Reply:
Ah, my bad